### PR TITLE
add osifeso oluwaseun

### PR DESCRIPTION
**Pull request by sheywenko**

### CODE REVIEW

**Comment by Oluwasetemi** [comment]
update the code by pulling from upstream and pushing to origin main

**Comment by Succynice** [comment]
please can you update your code by pulling the latest changes from the upstream.

```console
git pull upstream main

git push origin main

```

**Comment by sheywenko** [comment]
Alright

**Comment by sheywenko** [comment]
I just did that.
am i suppose to create another pr?

File: names.txt [hunk 1]
```diff
@@ -12,6 +12,7 @@ Pelumi Olukayode
 Ejiroghene Laurel Dafe
 Adeyomola Kazeem
 Kelechi Joanna Ojelabi
+Osifeso Oluwaseun
 Austin Wopara
```
Comment (oseunabiola) [inline names.txt:16]: Remove duplicate name

**Comment by oseunabiola** [comment]
> I just did that. am i suppose to create another pr?

You don't have to create a new PR. Pull, effect required changed and push again.

File: names.txt (hunk 1 above)
Comment (sheywenko) [reply names.txt:16]: I didn't see any

**Pull request closed without merging**